<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowLocando</class>
 <widget class="QMainWindow" name="MainWindowLocando">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Locar veiculo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>17</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Placa do veiculo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>45</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome do cliente:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>73</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>101</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor da locação:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>129</y>
      <width>68</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kilometragem:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlacaLocacao">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>17</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomeClienteLocacao">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>45</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDataLocacao">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>73</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValorLocacao">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>101</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKmInicial">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>129</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSituacaoLocacao">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>156</y>
      <width>57</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Locado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSalvarLocacao">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetGridVeiculos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>661</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Marca</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Modelo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Placa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Kilometragem</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor de compra</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Situação</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowLocando</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
